CHANGELOG/CHANGELOG-3.5.md

```diff
@@ -4,6 +4,13 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 
 <hr>
 
+## v3.5.6 (TBD)
```

Comment:
@serathius Do you plan to release 3.5.6 or 3.6.0 in the next step? 3.5.6 seems better if there are enough bug fixes.